ansible/roles/development/tasks/java.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: install java
+  pacman: 
+    name: jdk11-openjdk
```

Comment:
I'd like to remove any java related Installations with fixed versions in the automation part, as this should be the user's choice whether to install it or not and what version to use specifically